mptt/static/mptt/draggable-admin.js

```diff
@@ -402,7 +402,11 @@ django.jQuery(function($){
                 expandOrCollapseNode(treeNode(storedNodes[i]));
             }
         } else {
-            collapseTree();
+            if (DraggableMPTTAdmin.expandTreeByDefault) {
+                expandTree();
```

Comment:
You are right. I will remove the `expandTree()` clause.

The other point is that I thought you would not like to change the default behaviour of the class so that one have to decide to use this new feature to expand the tree on first visit. But then I recommend to use a flag that defaults to `False` so that one feels like opting in to a feature instead of opting out to a feature which is activated by default. I would always recommend to use flags that defaults to `False` in every programming context because doing this consequently reduces misunderstandings and silly bugs (a wise men thaught me once). Of course here it would not be a bug if we use `collapse_tree_by_default = True` instead of `expand_tree_by_default = False`. What about `dont_collapse_tree_on_first_visit = False`? (No problem if you don't like this ... :-))

So your project -> your choice ... ;-)